packages/babel-core/test/parse.js

```diff
@@ -28,4 +28,32 @@ describe("parse", function() {
     });
     expect(JSON.parse(JSON.stringify(result))).toEqual(output);
   });
+
+  it("should work", function() {
+    const program = "const identifier = 1";
+    const node = parse(program, {
+      filename: fixture("input.js"),
+      cwd: fixture(),
+    });
+    expect(node.program.body[0].declarations[0].id.name).toBe("identifier");
+    expect(node).not.toBeNull();
+    transformFromAst(node, program, {
+      filename: fixture("input.js"),
+      cwd: fixture(),
+      plugins: [
```

Comment:
this inline plugin forces the AST to change. any ideas on how to write it more succinctly?

Reply:
You can replace `function() { return` with `() => (`, but it is not really important

Reply:
Yup. I was wondering if you folks already have some established helper or something for this kind of stuff.